docs/extend/plugins.md

```diff
@@ -60,6 +60,8 @@ The following plugins exist:
 * The [Netshare plugin](https://github.com/gondor/docker-volume-netshare) is a volume plugin
   that provides volume management for NFS 3/4, AWS EFS and CIFS file systems.
 
+* The [OpenStorage Plugin] (https://github.com/libopenstorage/openstorage) is a cluster aware volume plugin that provides volume management for file and block storage solutions.  It implements a vendor neutral specification for implementing extensions such as CoS, encryption, and snapshots.   It has example drivers based on FUSE, NFS, NBD and EBS to name a few.
```

Comment:
The space between the `[]` and `()` parts of the link needs to be removed - I think it won't work well in Hugo (our publishing tool)

otherwise LGTM